src/utils.jl

```diff
@@ -0,0 +1,12 @@
+#
+"""
+We should define `sparse` for these types in `SciMLBase` instead,
+but that package doesn't know anything about sparse arrays yet, so
+we'll introduce a temporary work-around here.
+"""
+function _sparse end
+
+_sparse(L) = sparse(L)
+_sparse(L::SciMLMatrixOperator) = _sparse(L.A)
+_sparse(L::SciMLScaledOperator) = L.λ * _sparse(L.L)
```

Comment:
What exactly is this doing? Why is it not just `SparseArrays.sparse`?

Reply:
preserved it from older code. it's easier to just extend `SparseArrays.sparse`

Reply:
will rm `_sparse` and do that